modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php

```diff
@@ -124,22 +124,22 @@ public function getWidgetTypes() {
     return [
       'chart' => [
         'name' => $this->t('Chart'),
-        'service' => 'charts',
+        'services' => ['charts', 'chart', 'racing'],
```

Comment:
This method is a public API, and hence, breaks BC. So we need to leave `service` in place with `charts`. If people use this and rely on `service`, we should not break that. And if they want both, they can update their code.

Reply:
@upchuk, should we call `service` parameter deprecated to enforce update it by 3rd parties and don't support the same parameters?

Reply:
Ah yes yes, i wanted to say that as well but forgot.